src/sentry/options/defaults.py

```diff
@@ -629,6 +629,10 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Rollout configuration for Arroyo ConfluentProducer
+# Controls the rollout of individual Kafka producers by name
+register("arroyo.producer.confluent-producer-rollout", default={}, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
I think you will have to provide the type here to specify whether this is a set or a dict. 
Otherwise people could expect the wrong type when they use the option
```
type=Dict,
```